packages/gestalt/src/TextArea.js

```diff
@@ -68,39 +82,46 @@ export default class TextArea extends React.Component<Props, State> {
     return null;
   }
 
-  handleChange = (event: SyntheticInputEvent<>) => {
-    if (event.target instanceof HTMLTextAreaElement) {
-      this.props.onChange({
-        event,
-        value: event.target.value,
-      });
+  handleChange = (event: SyntheticInputEvent<HTMLTextAreaElement>) => {
+    this.props.onChange({
+      event,
+      value: event.currentTarget.value,
```

Comment:
are we sure this is the same, `event.target.value` and `event.currentTarget.value`? i know there was once a `currentTarget` `target` change in pinboard and it caused a rollback/hotfix